webhook-specification/v1.0/connections.yaml

```diff
@@ -59,57 +74,72 @@ components:
           example: 10
         connections:
           $ref: '#/components/schemas/Connections'
-        followers:
-          $ref: '#/components/schemas/Followers'
         graphKeypair:
           $ref: '#/components/schemas/GraphKeyPair'
     Connections:
-      type: array
-      items:
-        type: object
-        properties:
-          DSNPId:
-            type: integer
-            format: int64
-            example: 1024
-          privacy:
-            type: string
-            example: "Public"
-            enum:
-              - Public
-              - Private
-          connectionType:
-            type: string
-            example: "Follow"
-            enum:
-              - Follow
-              - Friendship
+      type: object
+      properties:
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/Connection'
+        pageNumber:
+          type: integer
+        pageSize:
+          type: integer
+        total:
+          type: integer
       xml:
         name: connections
-    Followers:
-      type: array
-      items:
-        type: object
-        properties:
-          DSNPId:
-            type: integer
-            format: int64
-            example: 2048
-          privacy:
-            type: string
-            example: "Public"
-            enum:
-              - Public
-              - Private
+      example:
+        data:
+          - DSNPId: 1024
+            privacy: "Public"
```

Comment:
lets make it `privacyType` to be in accordance with grapsdk models

Reply:
Done!